weave-js/src/components/Panel2/ChildPanel.tsx

```diff
@@ -630,6 +631,8 @@ export const ChildPanel: React.FC<ChildPanelProps> = props => {
                       <IconOverflowHorizontal />
                     </IconButton>
                   }
+                  onOpen={() => setIsMenuOpen(true)}
```

Comment:
This pattern edges on violating the single source of truth principle of controlled and uncontrolled components: https://react.dev/learn/sharing-state-between-components#controlled-and-uncontrolled-components.

Typically if you are going to manage the open state of a popup, you want to pass it in as well. This Popup uses Semantic's popup. See https://react.semantic-ui.com/modules/popup/#usage-controlled for details on this. 

In short, this will work, but there is a possibility of getting out of sync. Consider allowing caller to make this controlled by passing the popup state into the popup directly.